src/scripts/main.js

```diff
@@ -1,3 +1,336 @@
+  const heading = document.createElement('h2');
+  const paragraph = document.createElement('p');
+
+  divElement.classList.add('notification', type);
+  divElement.setAttribute('data-qa', 'notification');
+
+  heading.classList.add('title');
+  heading.textContent = title;
+  paragraph.textContent = description;
+
+  divElement.appendChild(heading);
+  divElement.appendChild(paragraph);
+
+  document.body.appendChild(divElement);
+
+  setTimeout(() => {
+    divElement.style.display = 'none';
+  }, 2000);
+};
+// #endregion
+// #region add class active
+let previousElement = null;
+
+document.addEventListener('click', (even) => {
+  const parentElement = even.target.parentElement;
+
+  if (tbody.contains(parentElement)) {
+    if (previousElement && previousElement !== parentElement) {
+      previousElement.classList.remove('active');
+    }
+
+    parentElement.classList.add('active');
+    previousElement = parentElement;
+  }
+});
+// #endregion
+// #region form
+
+const form = document.createElement('form');
+const existingElement = document.querySelector('script');
+
+form.classList.add('new-employee-form');
+
+const labelsAndInputs = [
+  {
+    label: 'Name:',
+    type: 'text',
+    name: 'name',
+    data: 'name',
+  },
+  {
+    label: 'Position:',
+    type: 'text',
+    name: 'position',
+    data: 'position',
+  },
+  {
+    label: 'Age:',
+    type: 'number',
+    name: 'age',
+    data: 'age',
+  },
+  {
+    label: 'Salary:',
+    type: 'number',
+    name: 'salary',
+    data: 'salary',
+  },
+];
+
+// Додаємо лейбли та інпуты до форми
+labelsAndInputs.forEach((item) => {
+  const label = document.createElement('label');
+  const input = document.createElement('input');
+
+  label.textContent = item.label;
+  input.type = item.type;
+  input.name = item.name;
+  input.required = true;
+  input.setAttribute('data-qa', item.data);
+
+  label.appendChild(input);
+  form.appendChild(label);
+});
+
+// Створюємо новий елемент <select>
+const selectLabel = document.createElement('label');
+const select = document.createElement('select');
+const cities = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco',
+];
+
+select.name = 'city';
+selectLabel.textContent = 'Office:';
+select.required = true;
+
+cities.forEach((city) => {
+  const option = document.createElement('option');
+
+  option.value = city;
+  option.textContent = city;
+  select.appendChild(option);
+});
+
+const AgeLab = form.querySelector('label input[name="age"]').parentElement;
+
+selectLabel.appendChild(select);
+
+form.insertBefore(selectLabel, AgeLab);
+
+// Створюємо кнопку
+const submitButton = document.createElement('button');
+
+submitButton.type = 'submit';
+submitButton.textContent = 'Save to table';
+form.appendChild(submitButton);
+
+document.body.insertBefore(form, existingElement);
+
+// #endregion
+// #region check form
+document
+  .querySelector('.new-employee-form')
+  .addEventListener('submit', (even) => {
```

Comment:
There is a typo in the parameter name 'even'. It should be 'event' to follow standard naming conventions for event handlers.